test/server/cards/03_TWI/units/EncouragingLeadership.spec.ts

```diff
@@ -0,0 +1,40 @@
+describe('EncouragingLeadership', function () {
```

Comment:
nit: our convention would be to write this as `'Encouraging Leadership'` with a space